pbjson-test/build.rs

```diff
@@ -23,13 +23,51 @@ fn main() -> Result<()> {
         .extern_path(".google.protobuf", "::pbjson_types")
         .extern_path(".test.external", "crate")
         .bytes(&[".test"])
-        .compile_protos(&proto_files, &[root])?;
+        .compile_protos(&proto_files, &[&root])?;
 
-    let descriptor_set = std::fs::read(descriptor_path)?;
+    let descriptor_set = std::fs::read(&descriptor_path)?;
     pbjson_build::Builder::new()
         .register_descriptors(&descriptor_set)?
         .extern_path(".test.external", "crate")
         .build(&[".test"])?;
 
+    println!("Build the first part");
+    let skipped_path = PathBuf::from(env::var("OUT_DIR").unwrap()).join("skipped/");
+
+    if !skipped_path.exists() {
+        std::fs::create_dir_all(&skipped_path)?;
+    }
+
+    println!("Putting stuff in: {:?}", skipped_path);
+
+    prost_build::Config::new()
+        .file_descriptor_set_path(&descriptor_path)
+        .compile_well_known_types()
+        .out_dir(&skipped_path)
+        .extern_path(".google.protobuf", "::pbjson_types")
+        .extern_path(".test.external", "crate")
+        .bytes(&[".test"])
+        .compile_protos(&proto_files, &[&root])
+        .map_err(|e| {
+            println!("Failed to build the protos..");
+
+            e
+        })?;
+
+    pbjson_build::Builder::new()
```

Comment:
I'm not a massive fan of the duplicated build options, with different output paths, perhaps we could make `skip_unknown_fields` take a type prefix like prost-build has for `bytes`, `btree_map`, etc...

Reply:
I'm not familiar with those tests, will definitely look at it and give it a try, as I'm not that big fan of this brain-dead duplication either.

Reply:
Can you point out to an example? I went on the prost-build, but not sure to what part you mean, or how could that prevent the duplication.

Reply:
Yeah coming back to this I'm not sure what I meant here :sweat_smile: